lib/awful/widget/layoutlist.lua

```diff
@@ -0,0 +1,442 @@
+----------------------------------------------------------------------------
+--- Display the available client layouts for a screen.
+--
+-- This is what the layoutlist looks like by default with a vertical layout:
+--
+--@DOC_awful_widget_layoutlist_default_EXAMPLE@
+--
+-- In the second example, it is shown how to create a popup in centered in
```

Comment:
`how to create a popup in the center of the screen`
or
`how to create a popup centered in the screen`

Reply:
Done